pkg/operator/ceph/cluster/mgr/mgr_test.go

```diff
@@ -133,10 +133,6 @@ func validateStart(t *testing.T, c *Cluster) {
 		assert.Equal(t, "my-priority-class", d.Spec.Template.Spec.PriorityClassName)
 		if c.spec.Mgr.Count == 1 {
```

Comment:
Remove this condition so we always assert on the next line that there is one container
```suggestion
```

Reply:
fixed